src/index.html

```diff
@@ -8,7 +8,43 @@
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body class="page">
+    <header class="header">
+      <a href="#logo" class="logo">
```

Comment:
```suggestion
      <a href="#" class="logo">
```
you don't want the logo to route to the `#logo` section 😉

Reply:
thank you :)